File: client/iih_stub/iih.ci_stub/iih.ci.mr/i/ICiOrdMrService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using xap.mw.core.data;

namespace iih.ci.iih.ci.mr.i
{
    public interface ICiOrdMrService
    {
        /// <summary>
        /// 刷新门诊处置数据到病历
        /// </summary>
        /// <param name="id_ent"></param>
        /// <param name="code_entp"></param>
        /// <returns></returns>
        Dictionary<string, object> GetOrderMrDtoFlushList(String id_ent, String code_entp);
	
	
        /// <summary>
        /// 刷新门诊诊断到病历
        /// </summary>
        /// <param name="id_ent"></param>
        /// <returns></returns>
	      String GetDiagList(String id_ent) ;
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/i/external/ICiOrdMrService.java
package iih.ci.ord.i.external;

import iih.ci.ord.ciorder.d.CiOrderDO;
import iih.ci.ord.dto.ordermr.d.OrderMrDto;
import xap.mw.core.data.BizException;
import xap.mw.core.data.FMap2;

/**
 * 临床提供给病历的对外接口
 * 
 * @author HUMS
 *
 */
public interface ICiOrdMrService {
	/**
	 * 处置手动刷新到病历 by yzh 2017-06-06 09:56:54
	 * 
	 * @param id_ent
	 * @param code_entp
	 * @param orders
	 * @return
	 * @throws BizException
	 */
	public abstract FMap2 getOrderMrDtoFlushList(String id_ent, String code_entp) throws BizException;
	
	/**
	 * 诊断刷新到病历 by yzh 2017-06-06 11:30:15
	 * @param id_ent
	 * @return
	 * @throws BizException
	 */
	public abstract String getDiagList(String id_ent) throws BizException;
}
